src/public/js/index.js

```diff
@@ -0,0 +1,28 @@
+import getTimeRemaining from './countdown';
+
+const initializeClock = (minutes) => {
+  document.getElementById('start').style.display = 'none';
+  document.getElementById('stop').style.display = 'block';
+  const endtime = Date.parse(new Date()) + (minutes * 60 * 1000); // number of miliseconds until countdown finishes
```

Comment:
`Date.parse(new Date())` is exactly the same as `Date.now()`

I think it is more readable to instead use `Date.now()`, also more consistent